package.json

```diff
@@ -89,6 +90,9 @@
     "modulePathIgnorePatterns": [
       "<rootDir>/example/node_modules",
       "<rootDir>/lib/"
+    ],
+    "setupFiles": [
+      "./src/__mocks__/VideoCompressor.ts"
```

Comment:
Since native modules aren't available during jest runs (it stays strictly in the javascript sphere), references to the native compressors should be mocked during tests.  Currently, jest is configured in the `package.json` file, but it could also be moved elsewhere per https://jestjs.io/docs/configuration